### PR TITLE
Make the Dockerfile.rocm to support the internal base docker build

### PR DESCRIPTION
**Pull request by WBobby**

### CODE REVIEW

File: tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm [hunk 1]
```diff
@@ -8,9 +8,15 @@ COPY setup.packages.rocm.cs7.sh setup.packages.rocm.cs7.sh
 COPY builder.packages.rocm.cs7.txt builder.packages.rocm.cs7.txt
 RUN /setup.packages.rocm.cs7.sh /builder.packages.rocm.cs7.txt
 
+ARG GPU_DEVICE_TARGETS="gfx900 gfx906 gfx908 gfx90a gfx1030"
+ENV GPU_DEVICE_TARGETS=${GPU_DEVICE_TARGETS}
+
 # Install ROCM
 ARG ROCM_VERSION
-ENV ROCM_PATH=/opt/rocm-${ROCM_VERSION}
+ARG INTERNAL_INSTALL
```
Comment (jayfurmanek) [inline tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm]: Can we change "INTERNAL_INSTALL" to "CUSTOM_INSTALL"?

**Review by jayfurmanek — APPROVED** [review]
It looks good. thx Just one comment/suggestion.

File: tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm (hunk 1 above)
Comment (WBobby) [reply tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm]: Just update the variable name. Thanks.

**Comment by jayfurmanek** [comment]
retest gpu-non-pip-multi please

**Comment by WBobby** [comment]
retest gpu-non-pip-multi please

**Comment by WBobby** [comment]
retest  Ubuntu-GPU-multi

**Pull request merged**